src/test/resources/log4j2-test.xml

```diff
@@ -8,8 +8,8 @@
 	<Loggers>
 		<Logger name="kafka" level="warn"/>
 		<Logger name="org.apache.kafka" level="warn"/>
-		<Logger name="org.springframework.kafka" level="warn"/>
-		<Logger name="org.springframework.integration" level="warn"/>
+		<Logger name="org.springframework.kafka" level="debug"/>
+		<Logger name="org.springframework.integration" level="debug"/>
```

Comment:
I think we don't need this change after you have pushed a commit.

Reply:
Will fix.